config/spl-build.m4

```diff
@@ -524,9 +524,9 @@ dnl #
 AC_DEFUN([SPL_AC_DEBUG_LOCK_TRACKING], [
 	AC_ARG_ENABLE([debug-lock-tracking],
 		[AS_HELP_STRING([--enable-debug-lock-tracking],
-		[Enable detailed lock init/destroy tracking  @<:@default=no@:>@])],
+		[Enable detailed lock init/destroy tracking  @<:@default=yes@:>@])],
 		[],
-		[enable_debug_lock_tracking=no])
+		[enable_debug_lock_tracking=yes])
```

Comment:
This can be accomplished by adding the following line to your top most commit comment.

```
CONFIG_OPTIONS="--enable-debug-lock-tracking --enable-debug"
```